Sparkle/CheckLocalizations.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+
+let args = NSProcessInfo.processInfo().arguments
```

Comment:
Can you easily access getopt_long from Swift? :D

Not a requirement, just would be nice.